### ISSUE TITLE
Warning about property using a constructor for "type"

### ISSUE THREAD

**Issue by rob4acre** [canjs/can-define#331]
__How often can you reproduce it?__

- [x] Always
- [ ] Sometimes
- [ ] Rarely
- [ ] Unable
- [ ] I didn’t try

__Description:__

I am converting an application we are developing from v3 to v4. It seems ok so far, but I am getting a warning that I don’t know how to fix. I have a property defined like

```
export default DefineMap.extend({
      loadingThings: {
        type: 'boolean',
        default: true
      },

```
I see a warning in the console like:

> can-define: the definition for loadingThings on DefineMap uses a constructor for “type”. Did you mean “Type”?

I get this for every property that has a type defined. 

__Environment:__

| Software                   | Version
| -------------------- | -------
| can-define version   | 2.2.3 
| Browser                    | Chrome
| Operating system    | OSX El Cap

**Comment by justinbmeyer** [comment]
Thanks.  This was my mistake.  I'll have a fix out in a moment.

**Comment by justinbmeyer** [comment]
fixed and released as 2.2.4 https://github.com/canjs/can-define/releases/tag/v2.2.4

**Comment by rob4acre** [comment]
That's fixed it. Thanks for the great responsiveness!
